docs/notebooks/icnn_inits.ipynb

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
<p>Unnecessary braces in <code>(dataloader_source, ..., )</code> </p><p>In general, would prefer code cells to be formatted (black is ok if @marcocuturi is ok with it; wasn't able to find <code>yapf</code>-based  formatter for notebooks).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>